tests/unit/amo/components/TestAddon.js

```diff
@@ -746,6 +711,83 @@ describe(__filename, () => {
     expect(rootNode.querySelector('.AddonDescription')).toEqual(null);
   });
 
+  it("does not display a lightweight theme's show more card", () => {
```

Comment:
Isn't is specific to the add-on description? If so, I'd update the test description and find `.AddonDescription`.

Reply:
well it's the whole card:

```javascript
  return showAbout ? (
      <ShowMoreCard header={title} className="AddonDescription">
        <div
          className="AddonDescription-contents"
          ref={(ref) => {
            this.addonDescription = ref;
          }}
          {...descriptionProps}
        />
      </ShowMoreCard>
    ) : null;
```javascript

should I still be checking specifically for the description?

Reply:
Ah sorry if I was not clear. If we introduce a second `ShowMoreCard`, then we might have issues. You should target this specific `ShowMoreCard` _via_ its classname.

Reply:
ahh...okay, I see... thanks